### PR TITLE
Move body font size and line height to normalize and also fix the values

### PR DESCRIPTION
**Pull request by kof**
Closes https://github.com/webstudio-is/webstudio-builder/issues/1845

## Description

See issue

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/react-sdk/src/css/normalize.ts [hunk 1]
```diff
@@ -127,6 +127,14 @@ export const body = [
       value: `system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'`,
     },
   },
+  {
+    property: "fontSize",
```
Comment (TrySound) [inline packages/react-sdk/src/css/normalize.ts:131]: Why normalize? This is kinda our custom preset.

File: packages/react-sdk/src/css/normalize.ts [hunk 2]
```diff
@@ -127,6 +127,14 @@ export const body = [
       value: `system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'`,
```
Comment (TrySound) [inline packages/react-sdk/src/css/normalize.ts:127]: We need a way to select this normalized fallback from ui.

Comment (kof) [reply packages/react-sdk/src/css/normalize.ts:127]: I don't understand what you mean

Comment (kof) [reply packages/react-sdk/src/css/normalize.ts]: got it, normalized the font

File: packages/react-sdk/src/css/normalize.ts [hunk 3]
```diff
@@ -124,7 +124,7 @@ export const body = [
     property: "fontFamily",
     value: {
       type: "keyword",
-      value: `system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'`,
+      value: "Arial, sans-serif",
```
Comment (TrySound) [inline packages/react-sdk/src/css/normalize.ts:127]: https://github.com/sindresorhus/modern-normalize/issues/3

We should use system-ui as default and expand into that list to mame ux consistent

Comment (kof) [reply packages/react-sdk/src/css/normalize.ts:127]: yeah, its a no fly, even they didn't merge

**Pull request merged**